src/components/forms/SearchBar/SearchBar.tsx

```diff
@@ -83,7 +76,7 @@ const SearchBar: React.FC<SearchBarProps> = ({
   const [isInvalid, setInvalid] = useState<boolean>(false);
   const [query, setQuery] = useState<string>(defaultValue);
   const [typingTimeout, setTypingTimeout] = useState<number>(0);
-  const [isSearching, setIsSearching] = useState<boolean>(false);
+  const [isSearching, setIsSearching] = useState<boolean>(true);
```

Comment:
This change results in showing the loading icon by default
<img width="853" alt="Screen Shot 2021-10-01 at 11 22 24 AM" src="https://user-images.githubusercontent.com/60668607/135597284-f435510c-772f-4b75-b5de-2568adb9ce8a.png">

Reply:
damn, good catch, I used that for testing since we are missing a bunch of search bar visual tests and forgot to change it back

Reply:
@youssefssc Fixed